docs/setup/deployment/production/stand-alone.md

```diff
@@ -62,14 +147,18 @@ To begin with, set the `export SECURE_SSL_REDIRECT=false` to prevent SSL redirec
 
 Then use the following commands to test run the server:
 
-* `npm run build`
-* `python manage.py collectstatic --noinput`
-* `python manage.py createcachetable`
-* `python manage.py migrate --noinput`
-* `python manage.py clear_cache --cache=default --cache=wagtailcache`
-* `python manage.py createsuperuser`
-* `python manage.py wagtailsiteupdate server.domain apply.server.domain 80`
-* `python manage.py runserver` \(runs development server at [http://127.0.0.1:8000](http://127.0.0.1:8000)\)
+```console
```

Comment:
`console` stands for bash session. I believe the intension here is to have these as copy/paste shell script. Use `shell` marker instead?

Reply:
![Screenshot 2024-01-25 at 7  57 26@2x](https://github.com/HyphaApp/hypha/assets/236356/bf39be66-d3cb-4fa8-b6bc-c1f24e11fa02)